src/functionalTest/java/uk/gov/hmcts/dm/functional/MultiMediaUploadIT.java

```diff
@@ -60,8 +59,6 @@ public void mv1R1AsAuthenticatedUserICannotUploadNotWhitelistedMultiMediaFiles()
 
     @Test
     public void uploadWhiteListedWithPassword_then_fail() {
-        assumeTrue(toggleConfiguration.isPasswordcheck());
```

Comment:
Hi @larslnde,
We will still need this check to ensure the test would not fail if we are required to turn this feature flag back to false via flux.